pkg/promql/functions_test.go

```diff
@@ -42,8 +42,8 @@ func TestDeriv(t *testing.T) {
 	a := storage.Appender(context.Background())
 
 	metric := labels.FromStrings("__name__", "foo")
-	a.Add(metric, 1493712816939, 1.0)
-	a.Add(metric, 1493712846939, 1.0)
+	a.Append(0, metric, 1493712816939, 1.0)
```

Comment:
there are golang-ci lint failures here

Reply:
that's from upstream and we planned not to change them according to us. So left it.